### DISCUSSION TITLE
Open Help

### DISCUSSION THREAD

**Discussion by SatinderpalSingh** [Computer-Science-and-Engineering-GNDEC/Hackathon-1.1#20] [Q&A]
We all know that this is a FOSS event, and we allowed the teams to give/take help from any other team. So ask your question here if any team needs any help from community. All the other teams are expected to help others.

**Comment by SatinderpalSingh** [comment]
- Report progress every hours
- Code commits on regular basis
- Running of code committed above on gne?.gndec.ac.in server
- After about 18, hours, our team should give some task on the basis of their  progress / performance, which they need to finish.
- This task may carry more weightage than mile stones decided by them.
- These need to decide milestones and they need to report progress. also in term of meeting milestones.

**Comment by PriyankaJhamb** [comment]
We are facing issue in saving email domain and later on github we found a issue thread where some other people experienced the same but solution isn't given by anyone through that thread. If anyone knows how to fix it kindly please reply

Github Thread:https://github.com/frappe/erpnext/issues/14857

**Reply by hsrai** [reply]
@PriyankaJhamb 

Give more detail.

Where it is hosted? Does that has security certificate, to have domain with https?

Which domain are you using for email? gmail or yahoo or some different?

Read all discussion there, and tricks are there to save it, and many has reported that it worked, like:

> Yes, I was able to make gmail account working after finding out gmail security settings etc but yes, that one works but not the own domain. I have tested different email accounts hosted by different shared servers. It may be picky with some certificate issues?

**Reply by chirudeep37** [reply]
gmail.com
I've tried all that tricks sir but still doesn't get any result

**Reply by hsrai** [reply]
@chirudeep37 

Give detailed report? Which action produce what error, and what was expected.

Also address all my concerns.

**Reply by PriyankaJhamb** [reply]
Respected Sir, 

I have attached a word document file to explain about the problem which is coming when we are creating email domain or email account for backend work.
[
[email domain problem.docx](https://github.com/Computer-Science-and-Engineering-GNDEC/Hackathon-1.1/files/6123439/email.domain.problem.docx)
]



[https://docs.erpnext.com/docs/user/manual/en/setting-up/email/email-account](https://docs.erpnext.com/docs/user/manual/en/setting-up/email/email-account)

In this erpnext manual, it is clearly written that for gmail,  it is not needed to create domain. 
So, we are confused why it is saying that create your domain first.




Thankyou

**Reply by hsrai** [reply]
You are supposed to work on gne?.gndec.ac.in server allotted.

**Reply by hsrai** [reply]
Email Domain for gmail is available out-of-the-box and you can directly proceed to create an Email Account by following step 2 of https://docs.erpnext.com/docs/user/manual/en/setting-up/email/email-account

**Reply by hsrai** [reply]
Search for:

Email Account list

in your erpNext installation.

Edit or add new. Type in notepad youremail@gmail.com. copy it and paste it in "Email address".

**Reply by PriyankaJhamb** [reply]
Thankyou Sir, I have done it and have created the Email account but unable to send email from ERPnext.

**Reply by PriyankaJhamb** [reply]
> You are supposed to work on gne?.gndec.ac.in server allotted.

Sir, server is not working properly. So, we are working at the local host.
